<?xml version='1.0' encoding='UTF-8' standalone='no'?>
<section xmlns="http://docbook.org/ns/docbook" version="5.0" xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="_dir_60b0ebf1c1a2617d8575026292fbab5a" xml:lang="fr">
<title>Répertoire de référence de public/glpi/vendor/symfony/cache/Exception</title>
<indexterm><primary>Répertoire de référence de public/glpi/vendor/symfony/cache/Exception</primary></indexterm>
Graphe des dépendances de répertoires pour Exception:<para>
    <informalfigure>
        <mediaobject>
            <imageobject>
                <imagedata width="50%" align="center" valign="middle" scalefit="0" fileref="dir_60b0ebf1c1a2617d8575026292fbab5a_dep.svg"></imagedata>
            </imageobject>
        </mediaobject>
    </informalfigure>
</para>
<simplesect>
    <title>Fichiers    </title>
        <itemizedlist>
            <listitem><para>fichier <link linkend="_symfony_2cache_2_exception_2_cache_exception_8php">CacheException.php</link></para>
</listitem>
            <listitem><para>fichier <link linkend="_symfony_2cache_2_exception_2_invalid_argument_exception_8php">InvalidArgumentException.php</link></para>
</listitem>
            <listitem><para>fichier <link linkend="_symfony_2cache_2_exception_2_logic_exception_8php">LogicException.php</link></para>
</listitem>
        </itemizedlist>
</simplesect>
</section>
